cbmgmtx/mgmt.go

```diff
@@ -98,6 +108,33 @@ func (h Management) DecodeCommonError(resp *http.Response) error {
 	}
 }
 
+type InvalidArg struct {
+	field  string
+	reason string
+}
+
+func parseForInvalidArg(body string) InvalidArg {
+	var invArg InvalidArg
+	_, errs, ok := strings.Cut(body, `errors":{`)
+	if !ok {
+		return invArg
+	}
+
+	firstErr, _, ok := strings.Cut(errs, ",")
```

Comment:
Is there a comma even when the server returns us only one error? Also a bit iffy that the server could return us an error message containing a command too.